source/manual/business-readiness-publish-changes.html.md

```diff
@@ -0,0 +1,40 @@
+---
+owner_slack: "#govuk-platform-health"
+title: Publishing changes to the business readiness finder
+section: Business readiness finder
+layout: manual_layout
+parent: "/manual.html"
+important: true
+last_reviewed_on: 2019-01-18
+review_in: 3 months
+---
+
+The full journey for the  [business readiness finder][business-readiness-finder] is made up of the content items:
+
+1. The start page: [`/business-uk-leaving-eu`](https://www.gov.uk/business-uk-leaving-eu)
+2. The question pages: [`/prepare-business-uk-leaving-eu`](https://www.gov.uk/prepare-business-uk-leaving-eu)
+3. The finder: [`/find-eu-exit-guidance-business`](https://www.gov.uk/find-eu-exit-guidance-business)
+
+The content fo the start page can be updated in Mainstream Publisher as you would any other publisher.
+
+The content item for the questions pages doesn't actually contain any details. The titles of the questions are defined in a YAML file in [finder frontend](finder-frontend). However, the body of the question, for example the options if the question is a checkbox are read from the content item of the finder itself (content item 3 in the list). Finder frontend searches the content item for a facet "key" that matches the question, e.g. `sector_business_area` to find the question content.
```

Comment:
Is it worth explicitly stating that changing the YAML for this will therefore affect both the facets on the finder and the options in the Q&A?

Reply:
That's what I am trying to get across. Unsuccessfully it seems 😞